src/xenia/cpu/backend/x64/x64_emitter.cc

```diff
@@ -74,23 +74,32 @@ X64Emitter::X64Emitter(X64Backend* backend, XbyakAllocator* allocator)
       backend_(backend),
       code_cache_(backend->code_cache()),
       allocator_(allocator) {
-  if (cvars::use_haswell_instructions) {
+  if (cvars::x64_extension_mask & kX64EmitAVX2)
```

Comment:
A bit too late, but we use brackets for all conditionals/loops.

Reply:
Ah shit, I didn't even notice that. Probably should refactor it anyway and use function to do the bit setting to make the code less noisy.